source/PlayerInfo.cpp

```diff
@@ -2095,6 +2120,70 @@ map<string, string> PlayerInfo::GetSubstitutions() const
 
 
 
+// TODO: Switch from internally using conditions for Tribute to those functions
+// TODO: Properly connect this function to the dominated property of planets
```

Comment:
that's not done yet, is it, are you letting that for another PR?

Reply:
Good catch! I'll include this in this PR.